tests/test_flowdiagnosticstool.cpp

```diff
@@ -344,7 +344,7 @@ BOOST_AUTO_TEST_CASE (OneDimCase)
         const auto tof = fwd.fd.timeOfFlight();
 
         BOOST_REQUIRE_EQUAL(tof.size(), cas.connectivity().numCells());
-        std::vector<double> expected = { 0.5, 1.5, 2.5, 3.5, 4.5 };
+        std::vector<double> expected = { 0.5, 1.5, 2.5, 3.5, 0.0 };
```

Comment:
> I see it here as a test of the "local" feature of the solver (0 tof in start cells).

Not a problem.

> I should expand the test a bit to cover some more borderline cases.

We can do that at a later point.  For now I'm satisfied that this cset enables me to run the NORNE_ATW2013 case in a reasonable time.